CONTRIBUTING.md

```diff
@@ -13,8 +13,9 @@ provide everything you need to get started.
    GitHub account (only applicable to outside contributors).
 3. [Create a new Git branch][urls.create_branch].
 4. Make your changes.
-5. Add a one line summary of changes to the "unreleased" section of `CHANGELOG.md`.
-6. [Submit the branch as a pull request][urls.submit_pr] to the repo. A team member should
+5. Add and/or update unit tests to cover your changes.
```

Comment:
Thanks for adding this. I suggest a minor edit here, since we have several different ways to test features. Not all of them are "unit" tests.
```suggestion
5. Add and/or update tests to cover your changes.
```